app.js

```diff
@@ -1,3 +1,7 @@
+var count = 0;
+var correctAnswers =[];
```

Comment:
You want to have a space around your operators for readability. The array here should have a single space before it, after the assignment operator.